src/commands/ext.ts

```diff
@@ -3,41 +3,40 @@ import * as clc from "cli-color";
 
 import { Command } from "../command";
 import * as getProjectId from "../getProjectId";
+import { logPrefix } from "../extensions/extensionsHelper";
 import { listExtensions } from "../extensions/listExtensions";
 import { requirePermissions } from "../requirePermissions";
 import * as logger from "../logger";
+import * as utils from "../utils";
 
 module.exports = new Command("ext")
   .description(
     "display information on how to use ext commands and extensions installed to your project"
   )
-  .before(requirePermissions, ["deploymentmanager.deployments.get"])
-  .action((options: any) => {
-    const projectId = getProjectId(options);
-    const commands = [
-      "ext-configure",
-      "ext-info",
-      "ext-install",
-      "ext-list",
-      "ext-uninstall",
-      "ext-update",
+  .action(async (options: any) => {
+    // Print out help info for all extensions commands.
+    utils.logLabeledBullet(logPrefix, "list of extensions commands:")
+    const firebaseTools = require("../"); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
This is a circular dependency AND a linter violation, but after a bunch of searching, Bryan and I concluded that this was the best of a bunch of bad options.